src/sentry/notifications/utils/__init__.py

```diff
@@ -127,10 +127,22 @@ def summarize_issues(
     return rv
 
 
-def get_link(group: Group, environment: str | None) -> str:
+def get_link(
+    group: Group,
+    environment: str | None,
+    alert_type: str | None = None,
+    alert_timestamp: str | None = None,
+    alert_rule_id: str | None = None,
+) -> str:
     query_params = {"referrer": "alert_email"}
     if environment:
         query_params["environment"] = environment
+    if alert_type:
+        query_params["alert_type"] = alert_type
+    if alert_timestamp:
+        query_params["alert_timestamp"] = alert_timestamp
+    if alert_rule_id:
+        query_params["alert_rule_id"] = alert_rule_id
```

Comment:
@barkbarkimashark are we sending these fields to analytics? Wondering how this change helps with analytcs

Reply:
I believe our team is working on adding those fields to analytics:
https://www.notion.so/sentry/Tech-Spec-for-Error-Monitoring-KPI-Instrumentation-946ba8f0ce89488780af9d4668023163